griffon/services/core_queries.py

```diff
@@ -241,8 +241,8 @@ def async_retrieve_upstreams(self, purl):
 
 def process_component(session, c):
     """perform any neccessary sub retrievals."""
-    # we only need to resolve on REDHAT namespace components (for now)
-    if c.namespace == "REDHAT":
+    # only process provided components
+    if c.software_build is None:
```

Comment:
Why do we restrict this only on components with no software builds?

Reply:
existence of software builds is another way of determining if a component is root or not ... if it is not a root then we should process - if it is a root, there is no need to